### PR TITLE
feat: text field is now hidden when not logged in

### LINKED ISSUE #26: Only Render Text Filter After Logged In
The text filter in the toolbar doesn't seem to work very well until the full task details are loaded (usually requires being logged in due to increased limits.)

Perhaps it makes sense to hide it until they are logged in. That way it will be easier to see the login with GitHub button on mobile!

work ubq fi_ (2)

### PR DESCRIPTION
**Pull request by gentlementlegen**
Resolves #26

### CODE REVIEW

**Comment by gentlementlegen** [comment]
What are the changes of this PR:
- text is hidden while logged out, visible when logged in
- the tests were all broken due to the use of OAuth with Supabase, before we use to simply put the token in the localStorage, now the session is really retrieved from Supabase `auth`
- while this is a clear improvement, it broke the tests as in I just faked the local storage before. Now, there is a real OAuth workflow going on. Sadly, Oauth cannot be mocked.
- dummy account was created for this purpose, credentials given to @pavlovcik and @rndquu, should be added to the env `UBIQUIBOT_GITHUB_USERNAME` and `UBIQUIBOT_GITHUB_PASSWORD`

File: static/style/style.css [hunk 1]
```diff
@@ -579,6 +579,9 @@
     font-size: 12px;
     letter-spacing: 0.5px;
   }
+  #filters input[type="text"].hidden {
+    visibility: hidden;
```
Comment (0x4007) [inline static/style/style.css]:
```suggestion
    display: none;
```

Comment (0x4007) [reply static/style/style.css]: I did this because it will not create an empty space in the DOM, which seems to make more sense for this purpose.

**Comment by 0x4007** [comment]
I know that @rndquu changed the Cloudflare deploys. You have the final say on whether to approve and merge this pull!

**Review by rndquu — APPROVED** [review]

**Comment by rndquu** [comment]
Works fine

> I know that @rndquu changed the Cloudflare deploys.

I haven't yet updated all of the repos with updated build workflow but current PR works fine and can be merged

**Comment by 0x4007** [comment]
Seems like it's not on production I tried refreshing several times

**Comment by gentlementlegen** [comment]
> Seems like it's not on production I tried refreshing several times

Still the case so far, got news about this?

**Comment by 0x4007** [comment]
@rndquu might know best

**Comment by rndquu** [comment]
> > Seems like it's not on production I tried refreshing several times
> 
> Still the case so far, got news about this?

Working on it

**Pull request merged**